.env.example

```diff
@@ -1,3 +1,4 @@
 LBLCS_JDBC_URL=jdbc:postgresql://postgres-dennys:5432/postgres?user=postgres&prepareThreshold=0
 LBLCS_JDBC_PW=dev-password
 RIOT_API_TOKEN=
+RIOT_USE_STUBS=True
```

Comment:
Is this used to replace tournament calls with stub calls for a certain lil guy? :3